### ISSUE TITLE
Set up new enclosure at Herb03

### ISSUE THREAD

**Issue by RebekkaML** [NHMDenmark/Herbarium-Sheets-workstation#135]
After taking down the enclosure at Herb03 (#134) and transporting it to Aarhus University to solve the issues they experienced with lighting and vibration (#121), we need to order supplies and construct a new enclosure for Herb03 at NHMD.

What needs to be done is:

- [x] find out how much material for the enclosure we already have and what we need to order
- [x] order components (complete list further down)
- [x] build new base plate and small monitor arm in house @Gomismis 
- [x] saw and start to assemble the new enclosure at ZM
- [x] transport everything from ZM to Priorparken
- [x] set up enclosure in Priorparken

**Comment by RebekkaML** [comment]
The supplies for the new enclosure can be found in #61 and for the clamps in #62. 

Since we still have some scaffolding material left from the last enclosures, I will talk to @Gomismis tomorrow to plan how much exactly we still need to order.

**Comment by Gomismis** [comment]
We need to order 4 x 3000 mm Bosh Rexroth (4x4 cm) Aluminium profiles.

**Comment by Gomismis** [comment]
In combination with the materials already in our possession, we can cut them as shown in the attached cut plan, with minimal waste. 

[Cut plan for enclosure no. 3 (winter 2024).pdf](https://github.com/user-attachments/files/17396994/Cut.plan.for.enclosure.no.3.winter.2024.pdf)

**Comment by RebekkaML** [comment]
After talking to @Gomismis, we determined that in addition to the aluminum bars, we need 8 corner connectors (we currently have 4 left) and 12 T-connectors (we have 2) for the enclosure. This means we need to order 4x corner connector and 10x T-connector, plus maybe 1-2 more of each in case of problems.

**Comment by RebekkaML** [comment]
Today I and @Gomismis dismantled the enclosure at Herb03 and listed everything that we will take to Aarhus and therefore need to reorder / remake:

- [x] 4 x 3000 mm Bosh Rexroth (4x4 cm) Aluminium profiles (instead: 2x3m, 4x2m)
- [x] 4-6 corner connectors
- [x] 10-12 T-connectors
- [x] Light clamps
- [x] New arm to hold Monitor and keyboard at Herb03
- [x] velcro zipties
- [x] base plate for enclosure @Gomismis 
- [x] 3D-printed holder for small screen @Gomismis 
- [x] Thermometer for inside the enclosure
- [x] security wires for the lamps
- [x] clamps for fastening the base plate to the copy stand
- [x] new fabric for the curtains

**Comment by chelseagraham** [comment]
Today I picked up two packages from Goecker and RS COmponents. They are in 249 on top of the Nanlux lamp boxes.

**Comment by chelseagraham** [comment]
Today I picked up a long package from RS Components. It is in 249.

**Comment by RebekkaML** [comment]
I accidentally ordered the wrong length of aluminum bars from RS Components, 2m instead of 3m. We redid the calculations and decided that it would be easier to order 2 additional 3m bars than sending everything back and order the original 4 3m bars that we wanted. Once they arrive we will then have 4x 2m, 2x 3m and some leftovers from the last enclosure builds. I created a new cutting plan for this scenario: [CutPlan_Updated.pdf](https://github.com/user-attachments/files/17660080/CutPlan_Updated.pdf)

**Comment by RebekkaML** [comment]
I ordered the components for the main enclosure, these are the order confirmations:
Aluminum beams and connectors: [Rs Components_Faktura_194130139.pdf](https://github.com/user-attachments/files/17660239/Rs.Components_Faktura_194130139.pdf)
More aluminum beams: [RS Components_order of new aluminum bars.pdf](https://github.com/user-attachments/files/17660240/RS.Components_order.of.new.aluminum.bars.pdf)
Clamps to hold the lights: [Goecker_Faktura_1231412.pdf](https://github.com/user-attachments/files/17660242/Goecker_Faktura_1231412.pdf)

**Comment by RebekkaML** [comment]
I ordered the arm to hold monitor and keyboard and Molton fabric (and clamps) for the curtain.
Monitor / keyboard arm: [Dustin_ergotron arm confirmation.pdf](https://github.com/user-attachments/files/17747760/Dustin_ergotron.arm.confirmation.pdf)
Molton / clamps: [Thoman confirmation Molton.pdf](https://github.com/user-attachments/files/17747763/Thoman.confirmation.Molton.pdf)

**Comment by RebekkaML** [comment]
I ordered the velcro zipties, security wires for the lamps, thermometer and clamps for the base plate.

velcro zipties and thermometer: [avXperten_velcro zipties_digital termometer.pdf](https://github.com/user-attachments/files/17847805/avXperten_velcro.zipties_digital.termometer.pdf)
security wires: [thomann_safety wires_confirmation.pdf](https://github.com/user-attachments/files/17847814/thomann_safety.wires_confirmation.pdf)
clamps: [avXperten_safety clamps.pdf](https://github.com/user-attachments/files/17847806/avXperten_safety.clamps.pdf)

With this, all components have been ordered and should arrive soon. What is left is building the things we can make in house, which @Gomismis will take care of. Then, we will assemble the enclosure as far as necessary and transport it to Priorparken.

**Comment by RebekkaML** [comment]
The Aluminum bars were sawed to the right size on 10.01.2025.
@Gomismis has ordered a car and will transport all necessary components from ZM to Priorparken on Mo, 20.01.2025.
On Tu, 21.01.2025 Matilde and I will assemble the workstation at Priorparken. The last things that are still missing, the base plate and the arm for the small monitor, will be added as soon as possible so the station can be used again.

**Comment by Gomismis** [comment]
I have added all the fabric sides to the enclosure and it ready for the final set up.

**Comment by Gomismis** [comment]
[enclosure fabric.pdf](https://github.com/user-attachments/files/18617504/enclosure.fabric.pdf) Drawing of how to assemble and cut the fabric.

**Comment by RebekkaML** [comment]
I started setting up the PC and camera at HERB03, but the wireless keyboard we used to have for this workstation no longer works, because it was part of a mouse / keyboard set and the mouse (and receiver) are in Aarhus now. For setting it up I used a cable keyboard from priorparken, but this won't work for digitization because the cable is just barely lnog enough and would be very annoying.
Once we solved this issue and the base plate is finished, the station should be ready for use.

**Comment by chelseagraham** [comment]
Today, 5 February 2025, @RebekkaML ordered a keyboard and mouse from KUIT no. 527182 for WORKHERB0003

**Comment by RebekkaML** [comment]
On friday, 7.2., I installed the new mouse and keyboard at the workstation and finished connecting the cables.

**Comment by chelseagraham** [comment]
So is this now ready for use?

**Comment by RebekkaML** [comment]
No, it still needs the base plate and the arm for the small monitor.

**Comment by chelseagraham** [comment]
@Gomismis when do you anticipate that you will have time to make the baseplate in the shop, 3D print the small monitor arm, transport them to Priorparken and then set them up?

**Comment by Gomismis** [comment]
Aren't they both at the station already? I did that a few weeks ago.

**Comment by RebekkaML** [comment]
Yes, they are there, they only need to be assembled and then the workstation is finished. If you tell me how to do it I could finish assembly when I'm at Priorparken on friday, @Gomismis.

**Comment by RebekkaML** [comment]
I covered the base plate in tape today and set the workstation up as far as I could.
But there were several things I could not find in Priorparken:

- the double sided tape for attaching the angle to the base plate
- something to attach the base of the small monitor arm to the T-connector (screwdriver bits)
- the small screen

I think the small screen and maybe also the double sided tape could still be in the box from Aarhus in Chelsea and Matildes office, I don't know where we have a fitting screwdriver bit or what we used to attach the small screen the last times.

I attached the angle with normal tape for now so I could start testing the light and taking some images.

@Gomismis do you know where the missing things are? Can you finish the workstation next week? Otherwise I could do it when I'm there on Friday.

**Comment by RebekkaML** [comment]
Matilde installed the arm for the small monitor on Thursday, 20.02 and I finished the enclosure on 21.02 by setting up the small monitor and fixing the angle to the base plate with double sided tape. I adjusted the lighting and took some test images, which Chelsea approved. 
The workstation is now fully functional and I started using it for regular imaging that day.
